### DISCUSSION TITLE
How to show widget only on selected sites?

### DISCUSSION THREAD

**Discussion by poozon84** [templaza/astroid-framework#560] [Q&A] [answered]
I would like to show astroid widget ( for ex. Articles ) only on selected sites ( for ex. HomePage ). Is there a way to do it?

**Comment by sonvnn** [comment, marked as answer]
At the moment, you have to create 2 templates one includes Article widget assign to Home page and one without Article widget assign to Blog page.

I will consider a way to assign widget to menu item in the future.

Thanks and Best regards,
Sonny

**Comment by poozon84** [comment]
Ok thanks, for now i used:
body:not(.start-alias) .astroid-articles { display:none; }
and it works well ( start is my homepage menu ) but articles code is downloaded on each page anyway even if not visible so not ideal way.

**Reply by sonvnn** [reply]
Yes, It is not good idea. You should follow my instruction. Create 2 templates 
1. A template includes Article Widget assign to Home Page
2. A template does not include Article Widget mark as default to apply to all others.

I think It's best solution right now.

**Comment by poozon84** [comment]
Hi, I can see you added "element visibility" feature. Can you explain how it works? It seems to not make any change for me when I change it to "current menu item only"

**Reply by sonvnn** [reply]
It looks like "Banner Visibility". That mean element will display on Category page only.

Category Page
![Screen Shot 2024-02-21 at 12 15 56](https://github.com/templaza/astroid-framework/assets/3204103/28ea484c-fc12-41cd-9a40-4948fd5378e1)

Article detail
![Screen Shot 2024-02-21 at 12 16 12](https://github.com/templaza/astroid-framework/assets/3204103/db9c43fa-ae2a-435c-aa4b-097199da02c0)

Best Regards,
Sonny
